pkg/minikube/out/out.go

```diff
@@ -126,12 +126,12 @@ func boxedCommon(printFunc func(format string, a ...interface{}), cfg box.Config
 
 // Boxed writes a stylized and templated message in a box to stdout using the default style config
 func Boxed(format string, a ...V) {
-	boxedCommon(String, defaultBoxCfg, "", format, a...)
+	boxedCommon(Stringf, defaultBoxCfg, "", format, a...)
 }
 
 // BoxedErr writes a stylized and templated message in a box to stderr using the default style config
 func BoxedErr(format string, a ...V) {
-	boxedCommon(Err, defaultBoxCfg, "", format, a...)
+	boxedCommon(Errf, defaultBoxCfg, "", format, a...)
```

Comment:
as above: not sure if this is correct

Reply:
I renamed the `Err` func to `Errf` but none of the logic changed, so this should be identical to before.

Reply:
hmm, i think it might've changed?

renamed Err to Errf now uses Outputf that uses fmt.Fprintf:
https://github.com/kubernetes/minikube/pull/19447/files#diff-b03808f35b53bc429e59e95cf4d9356db35223bff1d5c9fdebf4693b1746a088R299

and new Err uses Output that uses fmt.Fprint:
https://github.com/kubernetes/minikube/pull/19447/files#diff-b03808f35b53bc429e59e95cf4d9356db35223bff1d5c9fdebf4693b1746a088R276

Reply:
i was wrong here: Fprint does not add new line at the end, FprintLn does that and the change from eg Sprint to Sprintf was accompanied with comparative changes in Output -> Outputf, so i guess it should be ok